test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +36,20 @@ public void isAnyNull() {
         // confirms nulls inside the list are not considered
         List<Object> nullList = Arrays.asList((Object) null);
         assertFalse(Utils.isAnyNull(nullList));
+
+        // 2d array tests
+        ArrayList<ArrayList<String>> nullArrayList = new ArrayList<ArrayList<String>>(3);
+        for(int i = 0; i < 3; i++) {
+        	nullArrayList.add( i, new ArrayList<String>());
+        }
+        assertFalse(Utils.isAnyNull(nullArrayList));
+        
+        ArrayList<ArrayList<String>> notNullArrayList = new ArrayList<ArrayList<String>>(3);
+        for(int i = 0; i < 3; i++) {
+        	notNullArrayList.add( i, new ArrayList<String>());
+        	notNullArrayList.get(i).add("notNull");
+        }
+        assertFalse(Utils.isAnyNull(notNullArrayList));
```

Comment:
It could be worth abstracting out the method to set up of the object for the 2d array tests.